index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/normalize.css" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <script
+      src="https://kit.fontawesome.com/ee3d1963c1.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Check which line the right/left angle brackets are on throughout (for consistency)

Reply:
Having a look at my Visual Studio Code settings to 'fix' this.